.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Amikor olyan probléma merül fel, hogy valmelyik commit nincs meg local-ban és szükség lenne rá, akkor a git rebase master (vagy akár melyik branch neve, amelyikről szeretnénk commit-ot átvenni) parancsal lehet egyszerűen megoldani a problémát. De igy is működik természetesen.